tools/install.py

```diff
@@ -36,11 +36,12 @@ def run(target, user, config, toolchain, **kwargs):
 
     # Replace hostname with its IP address if the hostname is already known
     num_robots = 4
-    target = {
-        "{}{}".format(k, num): "10.1.1.{}".format(num)
-        for num in range(1, num_robots + 1)
-        for k, v in zip(("nugus", "n", "i", "igus"), [num] * num_robots)
-    }.get(target, target)
+    if target != "local":
+        target = {
+            "{}{}".format(k, num): "10.1.1.{}".format(num)
+            for num in range(1, num_robots + 1)
+            for k, v in zip(("nugus", "n", "i", "igus"), [num] * num_robots)
+        }.get(target, target)
```

Comment:
this hurts my brain lol. Not your changes, just this code :joy:

Reply:
Tell me about it :sweat_smile: